cafebabel/static/js/markdown-helper-position.js

```diff
@@ -0,0 +1,36 @@
+/* Add fixed position to markdown helper */
+const markdownHelper = document.querySelector('#markdown-helper')
+
+const editPageTextArea = document.querySelector('.textarea.bodytext'),
+  editPageTextAreaHeight = editPageTextArea.offsetHeight,
+  viewportHeight = Math.max(
+    document.documentElement.clientHeight,
+    window.innerHeight || 0
+  ),
+  markdownHelperCross = markdownHelper.querySelector('button')
+
+function isElementInViewport(el) {
+  const rect = el.getBoundingClientRect()
+  return (
+    rect.bottom >= 700 /* 700px to exclude footer height */ &&
+    rect.right >= 0 &&
+    rect.top <=
+      (window.innerHeight || document.documentElement.clientHeight) -
+        500 /* 500px after textarea enter viewport */ &&
+    rect.left <= (window.innerWidth || document.documentElement.clientWidth)
+  )
+}
+window.addEventListener('scroll', () => {
+  if (
+    isElementInViewport(editPageTextArea) &&
+    editPageTextAreaHeight >= viewportHeight
+  ) {
+    markdownHelper.classList.add('fixed')
+  } else {
+    markdownHelper.classList.remove('fixed')
+  }
```

Comment:
markdownHelper.classList.toggle('fixed', markdownHelper.classList.remove('fixed') && editPageTextAreaHeight >= viewportHeight)

Reply:
oh thanks ! didn't know this syntax. Is is supposed to be:
```
element.classList.toggle('class-to-toggle', condition)
```
hum nope don't think it is.. Can you explain?
Well anyway it doesn't work you way 😞

Reply:
@vinyll ?

Reply:
ok !

Reply:
Agree, that's a syntax detail, just fyi.

Reply:
😄